### PR TITLE
Remove broken header links in generated CLI docs

### PR DESCRIPTION
**Pull request by yorhodes**
### Description

Hack to remove header of each CLI command readme which contains broken links
               
### Tested

With `yarn preview`

### CODE REVIEW

File: packages/cli/generate_docs.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+set -e
+export COLUMNS=88
+yarn oclif-dev readme --multi --dir=../docs/command-line-interface
+yarn prettier --write ../docs/command-line-interface/*
+sed -i '' '/^- \[/d' ../docs/command-line-interface/*
```
Comment (eelanagaraj) [inline packages/cli/generate_docs.sh]: i think this line breaks the CI test on linux, [possible fix](https://stackoverflow.com/a/57766728)

**Review by eelanagaraj — COMMENTED** [review]
also I checked out + ran `yarn docs` and got more changes; might need to run + commit those files once more, but should be good to go after that!

**Comment by eelanagaraj** [comment]
@yorhodes I'm still getting a bunch of file changes and now errors when I try to run this locally. Maybe I'm doing something wrong? `yarn && yarn build --ignore docs`, then `yarn docs`, but running into a bunch of errors that look like: `[error] No parser could be inferred for file: ../docs/command-line-interface/*.md.bak`, and then a bunch of modified files. (Also tried `yarn docs` just within `/packages/cli` and still running into the same error).

**Comment by yorhodes** [comment]
> @yorhodes I'm still getting a bunch of file changes and now errors when I try to run this locally. Maybe I'm doing something wrong? `yarn && yarn build --ignore docs`, then `yarn docs`, but running into a bunch of errors that look like: `[error] No parser could be inferred for file: ../docs/command-line-interface/*.md.bak`, and then a bunch of modified files. (Also tried `yarn docs` just within `/packages/cli` and still running into the same error).

@eelanagaraj Ahh, just realized the backup files match the wildcard. Will push a fix.

**Review by eelanagaraj — APPROVED** [review]
LGTM

**Pull request merged**